backend/app/services/simulation_manager.py

```
@@ -1,529 +1,161 @@
-    TWITTER = "twitter"
-    REDDIT = "reddit"
-
-
 @dataclass
 class SimulationState:
-    """模拟状态"""
     simulation_id: str
     project_id: str
     graph_id: str
+    status: SimulationStatus = SimulationStatus.IDLE
+    created_at: str = field(default_factory=lambda: datetime.now().isoformat())
```

Ruff (0.15.11)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)